pandas/core/indexes/datetimes.py

```diff
@@ -1743,7 +1743,33 @@ def freq(self, value):
     is_year_start = _field_accessor(
         'is_year_start',
         'is_year_start',
-        "Logical indicating if first day of year (defined by frequency)")
+        """
+        Return a boolean indicating whether the date is the first day of the
```

Comment:
Same thing about the single line.

Reply:
General question here: for attributes, do we want to follow our rule of starting the summary line with a verb? 

I have the feeling that if all those attributes start with "Return", that's rather superfluous

Reply:
For example, this gives now a nice overview table with short explanations: http://pandas.pydata.org/pandas-docs/stable/api.html#time-date-components

Personally, I would say that if all those "The year of the datetime" become "Return the year of the datetime" etc, this is not necessarily an improvement

cc @datapythonista your view here is very appreciated

Reply:
I agree, I think that policy only makes sense to functions and methods, not to classes, properties or attributes.

If I'm not wrong, that is said in the documentation, and also the validation avoids the check for the infinitive verb.